lib/parse.js

```diff
@@ -58,65 +57,61 @@ const yaml = require('js-yaml');
  * >   }
  * > ]
  */
-
 module.exports = (input) => {
-  if (_.isEmpty(_.trim(input))) {
+  if (!input || !input.trim()) {
     return [];
   }
 
-  return _.compact(_.map(input.split(/\n\s*\n/), (device) => {
-    device = _.chain(device)
-      .split('\n')
-      .filter((line) => {
-        return /^(\s\s-\s)?[a-z]+:/i.test(line);
-      })
-      .map((line) => {
-        return line
-          .replace(/\\[^.\\]/g, (match, index, string) => {
-            const escapedCharacter = _.last(match);
-
-            if (string[index - 1] === '\\' || _.includes([
-              'b',
-              'f',
-              'n',
-              'r',
-              't',
-              'v'
-            ], escapedCharacter)) {
-              return match;
-            }
-
-            return escapedCharacter;
-          })
+  return input.split(/\n\s*\n/g)
+    .map((device) => {
+      device = device.split(/\r?\n/g)
+        .filter((line) => {
+          return /^(\s\s-\s)?[a-z]+:/i.test(line);
+        })
+        .map((line) => {
+          return line
+            .replace(/\\[^.\\]/g, (match, index, string) => {
+              const escapedCharacter = match[match.length - 1];
 
-          // Remove non printable ascii characters
-          // See http://stackoverflow.com/a/24229554
```

Comment:
I guess there's no reason to ditch the comment?

Reply:
That was unintentional. Fixed!